### PR TITLE
Updated versions in POM.

### PR DESCRIPTION
**Pull request by brandonforster**
Signed-off-by: Brandon Forster <brandonforster@gmail.com>

### CODE REVIEW

**Comment by tsconn23** [comment]
@JPWKU What's up with the non-completed checks? Are those still valid?

**Comment by JPWKU** [comment]
@tsconn23 Must have forgotten to disable those, doing it now.

File: pom.xml [hunk 1]
```diff
@@ -23,19 +23,19 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>org.edgexfoundry</groupId>
 	<artifactId>core-command</artifactId>
-	<version>0.5.0-SNAPSHOT</version>
+	<version>0.5.0</version>
```
Comment (JPWKU) [inline pom.xml]: @brandonforster we still want SNAPSHOT here on the actual app version

**Review by JPWKU — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (JPWKU) [reply pom.xml]: Also we should rev the master to 0.7.0-SNAPSHOT

**Comment by eb-oss** [comment]
Closing PR; project is being archived.

**Pull request closed without merging**